### PR TITLE
This patch series upgrades the operator-sdk and associated dependencies

### PR DESCRIPTION
**Pull request by bpradipt**
This series also fixes: #62

Tested on kubernetes version 1.24 and 1.25

### CODE REVIEW

File: bundle/manifests/cc-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,21 +12,16 @@ metadata:
...
+              "matchLabels": {
+                "node-role.kubernetes.io/worker": ""
+              }
+            },
             "config": {
               "cleanupCmd": [
                 "/opt/kata-artifacts/scripts/kata-deploy.sh",
                 "reset"
               ],
-              "cleanupNodeSelector": [
```
Comment (jensfr) [inline bundle/manifests/cc-operator.clusterserviceversion.yaml:20]: this is deleted because it's not in use any more?

Comment (jensfr) [reply bundle/manifests/cc-operator.clusterserviceversion.yaml:20]: looks like we have it hard coded in a few places. I guess that's okay, we'll fix it in a follow up

**Review by jensfr — APPROVED** [review]
looks good to me, thank  you @bpradipt

**Comment by jensfr** [comment]
tested it successfully, thanks for the quick fix!

File: bundle/manifests/cc-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (bpradipt) [reply bundle/manifests/cc-operator.clusterserviceversion.yaml:20]: Hmm, `make bundle` synced with the manifests under `config`. Let me just cross-check once to be sure. Thanks for catching this.

Comment (bpradipt) [reply bundle/manifests/cc-operator.clusterserviceversion.yaml:20]: @jensfr as you rightly mentioned the `cleanupNodeSelector` is no longer used and hence removed from bundle.

**Pull request merged**